mkosi/manifest.py

```diff
@@ -234,7 +234,21 @@ def has_data(self) -> bool:
         return len(self.packages) > 0
 
     def as_dict(self) -> Dict[str, Any]:
+        image = {
+            "distribution": self.args.distribution.name,
+            "release": self.args.release,
+            "architecture": self.args.architecture,
+            "format": self.args.output_format.name,
+        }
+        if self.args.image_id is not None:
+            image["name"] = self.args.image_id
+        else:
+            image["name"] = "image"
```

Comment:
```suggestion
        image["name"] = self.args.image_id or "image"
```
(this does has a semantic difference, though, `name` cannot be set to the empty string)